spec/06-expressions.md

```diff
@@ -261,13 +261,13 @@ If ´f´ has some value type, the application is taken to be equivalent to `´f
 i.e. the application of an `apply` method defined by ´f´. The value `´f´` is applicable to the given arguments if `´f´.apply` is applicable.
 
 
-Evaluation of `´f´(´e_1 , \ldots , e_n´)` usually entails evaluation of
-´f´ and ´e_1 , \ldots , e_n´ in that order. Each argument expression
-is converted to the type of its corresponding formal parameter.  After
-that, the application is rewritten to the function's right hand side,
-with actual arguments substituted for formal parameters.  The result
-of evaluating the rewritten right-hand side is finally converted to
-the function's declared result type, if one is given.
+Evaluation of `´f´(´e_1 , \ldots , e_n´)` first entails evaluation of
```

Comment:
What does it mean to first entail?
```suggestion
Evaluation of `´f´(´e_1 , \ldots , e_n´)` entails evaluation of
```

Also something weird with double spaces in this paragraph.
What an unusual PR!